### PR TITLE
Added lengths to crossbows in the game.

### PR DESCRIPTION
**Pull request by dchieh222**
Edited to include length for crossbows. Data from wikipedia as well as via descriptions within.

#### Summary

SUMMARY: Content "Adds lengths to crossbows."

#### Purpose of change

This adds lengths to crossbows, usually based on the prod length. Data taken mostly from comments within the original file, and converted to metric. Previously, crossbows had unrealistically small lengths.

#### Describe the solution

Crossbows now have accurately sized lengths.

#### Describe alternatives you've considered

I've considered using data from wikipedia and other online sites instead of what is in the comments, but in order to maintain consistency, kept with the current data. The current data is also consistent with real-world data.

The only one that had no data related to the original was the repeating crossbow. I used this one:

https://mandarinmansion.com/item/chinese-repeating-crossbow

#### Testing

I tested it in a game with a new world and a new character. Crossbows appear to operate as intended.

#### Additional context

https://mandarinmansion.com/item/chinese-repeating-crossbow for length of repeating crossbow

https://en.wikipedia.org/wiki/Bullet-shooting_crossbow for information on bullet crossbows

https://en.wikipedia.org/wiki/History_of_crossbows#Chinese_and_European_crossbows_in_comparison for further research if interested in the specific tiller and prod lengths of various historical crossbows.

### CODE REVIEW

File: data/json/items/ranged/crossbows.json [hunk 1]
```diff
@@ -298,6 +298,8 @@
     "ammo": [ "pebble" ],
     "weight": "1906 g",
     "volume": "1250 ml",
+    "longest_side": "86 cm",
+    "//": "34 inches, converted to metric.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:302]: I consider these comments redundant.

```suggestion
```

File: data/json/items/ranged/crossbows.json [hunk 2]
```diff
@@ -371,6 +375,8 @@
     "flags": [ "PRIMITIVE_RANGED_WEAPON" ],
     "skill": "rifle",
     "min_strength": 8,
+    "longest_side": "86 cm",
+    "//3": "34 inches, converted to metric.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:379]:
```suggestion
```

File: data/json/items/ranged/crossbows.json [hunk 3]
```diff
@@ -405,6 +411,8 @@
     "ammo": [ "bolt" ],
     "weight": "3500 g",
     "volume": "1750 ml",
+    "longest_side": "86 cm",
+    "//3": "34 inches, converted to metric.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:415]:
```suggestion
```

File: data/json/items/ranged/crossbows.json [hunk 4]
```diff
@@ -445,6 +453,8 @@
     "ammo": [ "bolt" ],
     "weight": "3000 g",
     "volume": "1750 ml",
+    "longest_side": "69 cm",
+    "//3": "27 inches, converted to metric.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:457]:
```suggestion
```

File: data/json/items/ranged/crossbows.json [hunk 5]
```diff
@@ -515,6 +526,8 @@
     "ammo": [ "bolt" ],
     "weight": "3628 g",
     "volume": "2500 ml",
+    "longest_side": "70 cm",
+    "//2": "Basically a pistol.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:530]:
```suggestion
```

File: data/json/items/ranged/crossbows.json [hunk 6]
```diff
@@ -336,6 +338,8 @@
     "ammo": [ "bolt" ],
     "weight": "900 g",
     "volume": "500 ml",
+    "longest_side": "70 cm",
+    "//2": "Basically a pistol.",
```
Comment (ZhilkinSerg) [inline data/json/items/ranged/crossbows.json:342]:
```suggestion
```

**Comment by dchieh222** [comment]
Accepted changes. Let me know if anything else is needed.

**Pull request merged**